src/library_webgpu.js

```diff
@@ -1917,7 +1917,7 @@ var LibraryWebGPU = {
       return 0;
     }
 
-    var data = _malloc(mapped.byteLength);
+    var data = _memalign(16, mapped.byteLength);
     HEAPU8.fill(0, data, mapped.byteLength);
```

Comment:
Unrelated but we have a `zeroMemory` JS helper that you can use instead of this line.